src/constants/chains.ts

```diff
@@ -1,43 +1,118 @@
 import arbitrumLogoUrl from 'assets/svg/arbitrum_logo.svg'
 import optimismLogoUrl from 'assets/svg/optimism_logo.svg'
 
-export enum SupportedChainId {
+export enum SupportedL2ChainId {
+  ARBITRUM_ONE = 42161,
+  ARBITRUM_RINKEBY = 421611,
+  OPTIMISM = 10,
+  OPTIMISTIC_KOVAN = 69,
+}
+
+export enum SupportedL1ChainId {
   MAINNET = 1,
   ROPSTEN = 3,
   RINKEBY = 4,
   GOERLI = 5,
   KOVAN = 42,
-  ARBITRUM_ONE = 42161,
-  ARBITRUM_RINKEBY = 421611,
-  OPTIMISM = 10,
-  OPTIMISTIC_KOVAN = 69,
+}
+export const SupportedChainId = {
```

Comment:
i don't love turning this into a const instead of an enum, since i don't think it works exactly the same as the old enum

can we achieve the same thing we want by having 

```
enum SupportedChainId { // all chains }

export type SupportedL2ChainId = SupportedChainId.ARBITRUM_ONE | SupportedChainId.OPTIMISM | ...

export SUPPORTED_CHAIN_IDS = [
 SupportedChainId.MAINNET,
 // all chains
] as const
```

Reply:
or just help me understand why this is better

Reply:
My approach isn't better, it just felt weird repeating the enum definition and creating two sources of truth for chainId constants. This way we lost the ability to use `SupportedChainId` as a type, but retained the ability to use a single reference point for all chainId values (`SupportedChainId.MAINNET`).